app/lua53/lnodemcu.c

```diff
@@ -436,11 +438,13 @@ static void eraseLFS(LFSflashState *F) {
 #ifdef LUA_USE_ESP
     if (*f == ~0 && !memcmp(f, f + 1, FLASH_PAGE_SIZE - sizeof(*f)))
       continue;
+    printf(".");
```

Comment:
Same as above re use of print. The code skips the erase where it is not need and this makes flashing of smaller LFS images a lot faster.   The dots are a pretty useful visual progress bar